forge/housekeeper/index.js

```diff
@@ -34,7 +34,7 @@ module.exports = fp(async function (app, _opts, next) {
         // Allow the housekeeper to be disabled - this allows the tests
         // to run without fear the housekeeper may fire off a task at the same
         // time.
-        if (!app.config.housekeeper) {
+        if (app.config.housekeeper === false) {
```

Comment:
This was my mistake, I thought we'd want `app.config.housekeeper` being null or undefined to trigger it being disabled too (since the previous logic was !== true).